### PR TITLE
changed time select order + added success message

### PR DESCRIPTION
**Pull request by edewit**
fixes: #445

### CODE REVIEW

**Comment by edewit** [comment]
now wraps the column title:
image

File: src/clients/initial-access/CreateInitialAccessToken.tsx [hunk 1]
```diff
@@ -78,13 +79,14 @@ export const CreateInitialAccessToken = () => {
           >
             <Controller
               name="expiration"
-              defaultValue=""
+              defaultValue={86400}
```
Comment (mfrances17) [inline src/clients/initial-access/CreateInitialAccessToken.tsx:82]: expiration needs a min val of 1, as per the UX review... i was able to select blank and 0, and type a negative value

**Review by mfrances17 — CHANGES_REQUESTED** [review]
Code looks good! One small issue, the rest seems to be working well to address the issues in 445

File: src/clients/initial-access/CreateInitialAccessToken.tsx (hunk 1 above)
Comment (edewit) [reply src/clients/initial-access/CreateInitialAccessToken.tsx:82]: This is behavior we need as per old console

Comment (mfrances17) [reply src/clients/initial-access/CreateInitialAccessToken.tsx:82]: ok

**Review by mfrances17 — APPROVED** [review]

**Review by jenny-s51 — APPROVED** [review]

**Pull request merged**